numba/np/random/random_methods.py

```diff
@@ -0,0 +1,349 @@
+import numpy as np
+
+from numba.core.extending import register_jitable
+
+from numba.np.random._constants import (UINT32_MAX, UINT64_MAX,
+                                        UINT16_MAX, UINT8_MAX)
+from numba.np.random.generator_core import next_uint32, next_uint64
+
+# All following implementations are direct translations from:
+# https://github.com/numpy/numpy/blob/7cfef93c77599bd387ecc6a15d186c5a46024dac/numpy/random/src/distributions/distributions.c
+
+
+@register_jitable
+def gen_mask(mask):
```

Comment:
this function is unused(?)

Reply:
Yes, I accidentally added it here, it's something from #5. Removed it in latest commit.